assets/src/edit-story/components/keyboardShortcutsMenu/index.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useRef, useState, useCallback, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useGlobalKeyDownEffect, useKeyDownEffect } from '../keyboard';
+import WithTooltip from '../tooltip';
+import Popup, { Placement } from '../popup';
+import { Keyboard as KeyboardShortcutsButton } from '../button';
+import ShortcutMenu from './shortcutMenu';
+
+function KeyboardShortcutsMenu({
+  placement = Placement.TOP_END,
+  spacing = { x: 0, y: 12 },
+  onMenuToggled,
+}) {
+  const anchorRef = useRef();
+  const isOpenBuffer = useRef(false);
+  const isOpenSetTimeout = useRef(0);
+
+  const [isOpen, setIsOpen] = useState(false);
+
+  const tryToggleMenu = useCallback(
+    (showMenu) => {
+      isOpenBuffer.current = showMenu;
+
+      // Stop previous timeout
+      window.clearTimeout(isOpenSetTimeout.current);
+
+      // Some toggle menu events can come in rapidly, to
+      // ensure that we don't just toggle the menu closed
+      // and then immediately open it, this little setTimeout
+      // will make sure only the last toggle will actually
+      // happen.
+      isOpenSetTimeout.current = window.setTimeout(() => {
+        setIsOpen(isOpenBuffer.current);
+        if (onMenuToggled) {
+          onMenuToggled(isOpenBuffer.current);
+        }
+
+        isOpenBuffer.current = false;
+      }, 10);
+    },
+    [onMenuToggled]
+  );
+
+  const toggleMenu = useCallback(
+    (_, showMenu) => {
+      tryToggleMenu(showMenu ?? !isOpen);
+    },
+    [tryToggleMenu, isOpen]
+  );
+
+  useGlobalKeyDownEffect('mod+/', toggleMenu, [toggleMenu]);
```

Comment:
Interesting... the shortcut works on my keyboard, but the `/` is the main key on my keyboard (with `?` being the alternate).  I'll investigate on how to get this shortcut working properly whether `/` is the main or alternate key.

Reply:
@barklund  Okay, just to test things out I changed the shortcut to `mod+:`, which mimics the situation you have with your keyboard in the following ways:

1.) In order for me to type `:` I have to type `Shift`+`;`.
2.) `Cmd`+`Shift`+`;` triggers some Chrome shortcut.

Given the above, I added in the `evt.preventDefault()` call as you suggested, and when I'm in the editor and I type `Cmd`+`Shift`+`;`, the menu opens up and the chrome shortcut doesn't get trigger 🎉 

I returned the shortcut to be `mod+/`, but it _should_ now hopefully work on your keyboard even with having to press the `Shift` key.  If it still doesn't... I may need to ask Sam for a more universal shortcut command combination to toggle the menu open/closed because maybe the Chrome shortcut it's triggering can't be overridden.